sw/device/examples/hello_usbdev/hello_usbdev.c

```diff
@@ -75,10 +75,23 @@ static void usb_receipt_callback_1(uint8_t c) {
   ++usb_chars_recved_total;
 }
 
+static void usb_send_str(char *s, usb_ss_ctx_t *ss) {
```

Comment:
Can you give these parameters more specific names? Also, please include a Doxygen comment on this helper.

Also, I think `s` can be `const char *`.

Reply:
Done.